projects/workshop/desafio_rick_and_morty/pubspec.yaml

```
@@ -0,0 +1,27 @@
+  go_router: ^14.2.0
+  cached_network_image: ^3.4.1
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^4.0.0
+
+flutter:
+  uses-material-design: true
+  assets:
+    - assets/images/logo.png
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)